guides/assets/stylesheets/main.css

```
@@ -729,12 +729,9 @@ div.important p, div.caution p, div.warning p, div.note p, div.info p {
 }
 
 /* When triple-clicking shell code, select only the command, not the prompt */
-code.language-shell-session span.token.command {
+code.highlight.console span.gp {
   display: inline-flex;
 }
```

Try this:

```suggestion
code.highlight.console span.w {
  display: table-cell;
}
```

Or, if `.w` shows up in more places and we need more specificity:

```suggestion
code.highlight.console span.gp + span.w {
  display: table-cell;
}
```

I tested in Firefox and Chromium, and triple-click selection from #39594 works for recognized commands (`BUILTINS`).  It doesn't work as well for non-recognized commands because Rouge doesn't add a `span` after the `.w` in that case, but that's probably okay.

---

Thanks! I've chosen the simpler first solution. I've grep-ed for `.w` and it only seems to be used for the console commands.